__tests__/jesttest.tsx

```diff
@@ -92,6 +94,55 @@ describe("LibraryContainer", function () {
     expect(libraryItem.prop('data').childItems[0].childItems[0].text).to.equal("Child1");
     expect(libraryItem.prop('data').childItems[0].childItems[1].text).to.equal("Child2");
   });
+   describe("Test event on libraryitem", function () {
```

Comment:
@ellensi Nesting of "describe" is allowed by the test framework. If we want to run specific type of tests into different groups it can be used. Right now i'm removing it here as we have only two tests. https://github.com/DynamoDS/librarie.js/pull/77